firebase-ai/src/testUtil/java/com/google/firebase/ai/JavaCompileTests.java

```diff
@@ -236,4 +270,62 @@ public void validateUsageMetadata(UsageMetadata metadata) {
       }
     }
   }
+
+  private void testLiveFutures(LiveModelFutures futures) throws Exception {
+    LiveSessionFutures session = futures.connect().get();
+    session
+        .receive()
+        .subscribe(
+            new Subscriber<LiveContentResponse>() {
+              @Override
+              public void onSubscribe(Subscription s) {
+                s.request(Long.MAX_VALUE);
+              }
+
+              @Override
+              public void onNext(LiveContentResponse response) {
+                validateLiveContentResponse(response);
+              }
+
+              @Override
+              public void onError(Throwable t) {
+                // Ignore
+              }
+
+              @Override
+              public void onComplete() {
+                // Also ignore
+              }
+            });
+
+    session.send("Fake message");
+    session.send(new Content.Builder().addText("Fake message").build());
+
+    byte[] bytes = new byte[] {(byte) 0xCA, (byte) 0xFE, (byte) 0xBA, (byte) 0xBE};
+    session.sendMediaStream(List.of(new MediaData(bytes, "image/jxl")));
+
+    FunctionResponsePart functionResponse =
+        new FunctionResponsePart("myFunction", new JsonObject(Map.of()));
+    session.sendFunctionResponse(List.of(functionResponse, functionResponse));
+
+    session.startAudioConversation(part -> functionResponse);
+    session.startAudioConversation();
+    session.stopAudioConversation();
+    session.stopReceiving();
+    session.close();
+  }
+
+  private void validateLiveContentResponse(LiveContentResponse response) {
+    // int status = response.getStatus();
+    // Assert.assertEquals(status, LiveContentResponse.Status.Companion.getNORMAL());
+    // Assert.assertNotEquals(status, LiveContentResponse.Status.Companion.getINTERRUPTED());
+    // Assert.assertNotEquals(status, LiveContentResponse.Status.Companion.getTURN_COMPLETE());
+    // TODO b/412743328 LiveContentResponse.Status inaccessible for Java users
```

Comment:
These will be changed to properties with #6910 (getters in java)

Reply:
Sounds good, I'll adjust these when I make breaking changes for java compat in a follow up all at once